New Fax Responder map + Overwatch Console sprites

noctyrnal opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
<!-- What did you change? -->
Changes the fax responder map to be more visually appealing and make faction rooms more distinct.
Gives OW Console new south and missing north sprites.

## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
Sovl

## Technical details
<!-- Summary of code changes for easier review. -->
barely any yaml and sprites

## Media
<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc). 
Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->
![image](https://github.com/user-attachments/assets/cd0c66d0-019d-4bf9-bab1-bec74732cb7d)
![image](https://github.com/user-attachments/assets/7855fd8a-84c4-44e3-aa8d-93c93d4184e3)
![image](https://github.com/user-attachments/assets/3c1f9d01-955d-4d34-905a-b1588fb5c20d)

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->
- [x] By submitting this code and/or assets, I confirm that I either own them or have provided the correct necessary licenses to use and distribute them. I agree to be fully responsible for any legal claims or issues arising from the use of these materials.

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->
<!--
:cl:
- add: Added fun!
- remove: Removed fun!
- tweak: Changed fun!
- fix: Fixed fun!
-->
:cl: noctyrnal
- add: Added new sprites for the OW console's south and north face.
- tweak: Changed the Fax Responder map to be more 'appealing' visually.

DrSmugleaf reviewed (dismissed):
no more curse of gigantic overwatch computer

noctyrnal commented:
> ```
> Error: [/Resources/Maps/_RMC14/admin_fax.yml] entities.3.entities.0.paused: Unexpected element
> Error: [/Resources/Maps/_RMC14/admin_fax.yml] entities.3.entities.0.mapInit: Unexpected element
> ```

fixed, also removed the fake Z and make the we-ya and press rooms more boxy

noctyrnal commented:
> > Don't forget to include changes from #6595
> 
> I can change the rest but idk about removing physics I remember sloth talking to me about how removing physics can cause weird shit to occur and lag

ok i checked and seems they're all already done

noctyrnal commented:
ok wait i lied let me fix
